content/numpy/concepts/built-in-functions/built-in-functions.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'Built in Functions'
+Description: 'NumPy has many built-in functions for mathematical operations and array manipulation.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
```

Comment:
Just one `paths/` link. Wouldn't `paths/data-science` be a good candidate for these entries?